src/compounds/RequestItem/RequestItem.jsx

```diff
@@ -6,8 +6,8 @@ import TextBox from '../../components/TextBox/TextBox'
 import Title from '../../components/Title/Title'
 import './request-item.scss'
 
-const RequestItem = React.forwardRef(({ backgroundColor, index, request }, ref) => {
-  const { createdAt, description, href, img, title, status, updatedAt } = request
+const RequestItem = React.forwardRef(({ backgroundColor, href, index, request }, ref) => {
```

Comment:
we need the converted version of `href` that comes from the forwardRef, not the object that we included in the request itself.